client/src/views/Cart.vue

```diff
@@ -13,13 +13,15 @@
             <th>Partnumber</th>
             <th>Description</th>
             <th>Qty</th>
+            <th></th>
           </tr>
         </thead>
         <tbody>
-          <tr v-for="row in NewItems" v-bind:key="row.PartId">
-            <td>{{row.PartId}}</td>
+          <tr v-for="row in rows" v-bind:key="row.PartId">
+            <td>{{row.partnumber}}</td>
             <td>{{row.description}}</td>
-            <td>{{row.qty}}</td>
+            <td><input class="item-qty" type="number" v-model='row.qty' @input='persistCart()' /></td>
+            <td><a @click="removeFromCart(row)">Remove</a></td>
```

Comment:
I kind of took over the table. This is mostly just to show two things: integration for removing items from the cart when clicking the link (but I think you knew how to do that), and (more importantly) how the quantity of an item can be updated. Setting the quantity field of the object as the model of an input will trigger any updates needed for anything observing the quantity value. After that the only thing that still needs to happen is saving the change in the cart, so that's added in also. There may be a way to set up a watch in the store to persist the cart whenever there's any change to the array or objects in it, but if there aren't any other interactions that it would be needed for then this is probably cleaner.